map/bookmark_manager.cpp

```diff
@@ -96,6 +103,115 @@ BookmarkManager::SharingResult GetFileForSharing(BookmarkManager::KMLDataCollect
   return {{categoryId}, std::move(tmpFilePath)};
 }
 
+bool addFileToArchive(std::string const & filePath, std::string const & fileInArchive, zip::File zip)
+{
+  zip::FileInfo zipInfo = {};
+  CreateTMZip(zipInfo.tmz_date);
```

Comment:
Why filling date is important/required?

Reply:
Without this setting dates for archived files looks weird:
![image](https://github.com/organicmaps/organicmaps/assets/128428520/304fc0ba-6847-4095-ac7d-6bfdd0a1e4d0)